src/providers/bitcoind.ts

```diff
@@ -130,19 +130,43 @@ export class BitcoindProvider implements Provider {
     return fees;
   }
 
+  /**
+   * Fetches min fee rate from the Bitcoind server.
+   *
+   * @returns A promise that resolves to the fetched min fee rate.
+   */
+  async getMinRelayFeeRate(): Promise<number> {
+    const getMempoolInfo = promisify(
+      this.rpc.getMempoolInfo.bind(this.rpc),
+    );
+
+    const response = await getMempoolInfo();
+    log.trace({ message: "getMempoolInfo", response: response.result });
+
+    const feeRate = response.result?.mempoolminfee;
+    if (!feeRate) {
+      log.error({ message: "Error getting mempool min fee" });
```

Comment:
I guess you could be a little bit more specific in this error to make it easier to find in the logs later.
Something like:
```suggestion
      log.error({ message: "Error getting mempool min fee, mempoolminfee not found" });
```

Reply:
Good point, I improved it here a90e18c76a09d7a9c5fb50d9e6b7488e36a24932